packages/react/src/progress-indicator/ProgressIndicatorItem.tsx

```diff
@@ -213,7 +237,17 @@ const progressIndicatorItemTextContainerDataAttr =
 const progressIndicatorItemTextDataAttr =
 	'data-agds-progress-indicator-item-text';
 
-const statusMap = {
+type IconColour = 'border' | 'selected' | 'success' | 'error';
```

Comment:
It would make sense for this type to be declared somewhere else that is more universally accessible. I'm not sure where that would be.

Reply:
It's an easy fix to move later if we want to. I'll resolve this conversation